packages/flame_svg/lib/svg_component.dart

```diff
@@ -7,18 +7,26 @@ import './svg.dart';
 /// Wraps [Svg] in a Flame component.
 class SvgComponent extends PositionComponent {
   /// The wrapped instance of [Svg].
-  Svg svg;
+  final Svg? svg;
+
+  /// Creates an [SvgComponent]
+  SvgComponent({
+    this.svg,
+    Vector2? position,
+    Vector2? size,
+    int? priority,
+  }) : super(position: position, size: size, priority: priority);
```

Comment:
What about scale and angle? And maybe there are more arguments on PositionComponent, on my phone currently.